patterns/1-initial/agile-innersource-dojo.md

```diff
@@ -0,0 +1,76 @@
+## Title
+
+Agile InnerSource Dojo
+
+## Patlet
+
+Organizations struggle with siloed knowledge and slow adaptation. Agile environments left unchecked can lead to redundant codebases feeding the desire to over-standardized practices that ultimately stifle innovation. Implementing an Agile InnerSource Dojo creates a structured learning environment which fosters improved collaboration, reduced waste, improved efficiency and promotes generative culture, supporting DevOps and scaled change initiatives.
```

Comment:
```suggestion
Organizations struggle with siloed knowledge and slow adaptation. Agile environments left unchecked can lead to redundant codebases feeding the desire to over-standardized practices that ultimately stifle innovation. Implementing an Agile InnerSource Dojo creates a structured learning environment which fosters improved collaboration, reduced waste, improved efficiency and promotes generative culture, supporting scaled change initiatives and other best practices such as DevOps.
```

Given DevOps isn't mentioned elsewhere, I wonder if it's better used as an example of other best practices rather then called out specifically? Unless there's a specific DevOps angle I'm missing, in which case it would be good to expand on this in the pattern.